### PR TITLE
Add issue badge in README.md

### PR DESCRIPTION
**Pull request by shivam-sharma7**
## Describe your changes
Add issue badge in README.md

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by shivam-sharma7** [comment]
image

File: README.md [hunk 1]
```diff
@@ -5,7 +5,8 @@
 </div>
 <br>
 
-[![Build](https://github.com/quarkiverse/quarkus-quinoa/workflows/Build/badge.svg)](https://github.com/quarkiverse/quarkus-quinoa/actions?query=workflow%3ABuild)
+[![Build](https://github.com/quarkiverse/quarkus-quinoa/workflows/Build/badge.svg)](https://github.com/quarkiverse/quarkus-quinoa/actions?query=workflow%3ABuild) 
+<a href="https://github.com/quarkiverse/quarkus-quinoa/issues"><img alt="GitHub issues" src="https://img.shields.io/github/issues/quarkiverse/quarkus-quinoa"></a>
```
Comment (melloware) [inline README.md:9]:
```suggestion
[![Issues](https://img.shields.io/github/issues/quarkiverse/quarkus-quinoa)](https://github.com/quarkiverse/quarkus-quinoa/issues) 
```

**Review by melloware — CHANGES_REQUESTED** [review]

**Comment by shivam-sharma7** [comment]
@melloware how to run this project locally?

**Comment by melloware** [comment]
What I do....

`mvn clean install`

Then you can `cd integration-tests` and run `mvn quarkus:dev`

**Comment by shivam-sharma7** [comment]
Thanks

**Comment by melloware** [comment]
yep `To build this project Maven 3.8.6 (or greater) is required. Please install it.`

Do `mvn --version` you will see your Maven version is less than 3.8.6.

**Comment by melloware** [comment]
oh yeah WSL makes it tougher but try this: https://qiita.com/Kain-Qiita/items/05bacbe6f94a8e4719af

**Comment by shivam-sharma7** [comment]
again error

```
Profile dev activated. Live Coding activated.
2023-05-02 22:33:46,435 INFO  [io.quarkus] (Quarkus Main Thread) Installed features: [cdi, quinoa, smallrye-context-propagation, vertx]
2023-05-02 22:33:46,609 DEBUG [io.qua.qui.QuinoaDevProxyHandler] (vert.x-eventloop-thread-2) Quinoa is forwarding: '/'
2023-05-02 22:33:46,683 INFO  [io.quarkus] (Quarkus Main Thread) quarkus-quinoa-deployment stopped in 0.023s
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 18.891 s - in io.quarkiverse.quinoa.test.devmode.ForwardedDevModeTest
[INFO] 
[INFO] Results:
[INFO]
[ERROR] Errors: 
[ERROR]   QuinoaPackageManagerLockfileDetectPNPMTest » Runtime java.lang.RuntimeException: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
        [error]: Build step io.quarkiverse.quinoa.deployment.QuinoaProcessor#prepareQuinoaDirectory threw an exception: java.lang.RuntimeException: Error in Quinoa while running package manager install command: pnpm.cmd install
        at io.quarkiverse.quinoa.deployment.packagemanager.PackageManager.install(PackageManager.java:45)
        at io.quarkiverse.quinoa.deployment.QuinoaProcessor.prepareQuinoaDirectory(QuinoaProcessor.java:107)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:568)
        at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:909)
        at io.quarkus.builder.BuildContext.run(BuildContext.java:282)
        at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
        at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2513)
        at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1538)
        at java.base/java.lang.Thread.run(Thread.java:833)
        at org.jboss.threads.JBossThread.run(JBossThread.java:501)

[INFO]
[ERROR] Tests run: 24, Failures: 0, Errors: 1, Skipped: 0
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Quarkus - Quinoa - Parent 999-SNAPSHOT:
[INFO]
[INFO] Quarkus - Quinoa - Parent .......................... SUCCESS [  3.667 s]
[INFO] Quarkus - Quinoa - Deployment - Testing ............ SUCCESS [  8.683 s]
[INFO] Quarkus - Quinoa - Runtime ......................... SUCCESS [  7.064 s]
[INFO] Quarkus - Quinoa - Deployment ...................... FAILURE [08:20 min]
[INFO] Quarkus - Quinoa - Testing ......................... SKIPPED
[INFO] Quarkus - Quinoa - Documentation ................... SKIPPED
[INFO] Quarkus - Quinoa - Integration Tests ............... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  08:40 min
[INFO] Finished at: 2023-05-02T22:33:47+05:30
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.0.0:test (default-test) on project quarkus-quinoa-deployment:
[ERROR]
[ERROR] Please refer to C:\Users\shivam sharma\Desktop\Open-Source\quarkus-quinoa\deployment\target\surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :quarkus-quinoa-deployment

```

**Comment by melloware** [comment]
That looks like some kind of bug `pnpm.cmd install` should not be trying to install the windows CMD file but the Linux `.sh` file.

**Comment by melloware** [comment]
Ahh on Linux you have to run this `npm install -g pnpm` once according to what the Linux GitHub actions do.

https://github.com/quarkiverse/quarkus-quinoa/blob/08698648abfc31a269639b4183dc6b6b2c48719d/.github/workflows/build.yml#L51-L52

**Comment by shivam-sharma7** [comment]
This time got new error

```
[INFO] No proxy was configured, downloading directly
[INFO] Unpacking /mnt/c/Users/shivam sharma/Desktop/Open-Source/quarkus-quinoa/integration-tests/target/quarkus-codestart-test/quarkus-quinoa-d7ee3bb9-8b8c-4033-ab60-3eb98338e11b/real-data/java/.quinoa/cache/node-v18.16.0-linux-x64.tar.gz into /mnt/c/Users/shivam sharma/Desktop/Open-Source/quarkus-quinoa/integration-tests/target/quarkus-codestart-test/quarkus-quinoa-d7ee3bb9-8b8c-4033-ab60-3eb98338e11b/real-data/java/.quinoa/node/tmp

applying codestarts...
📚 java
🔨 maven
📦 quarkus
📝 config-properties
🔧 dockerfiles
🔧 maven-wrapper
🚀 quinoa-codestart
[ERROR] Tests run: 2, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 605.213 s <<< FAILURE! - in io.quarkiverse.quinoa.it.QuinoaCodestartTest
[ERROR] io.quarkiverse.quinoa.it.QuinoaCodestartTest.testBuild  Time elapsed: 603.454 s  <<< ERROR!
java.lang.IllegalThreadStateException: process hasn't exited
        at java.base/java.lang.ProcessImpl.exitValue(ProcessImpl.java:521)
        at io.quarkus.devtools.testing.WrapperRunner.run(WrapperRunner.java:97)
        at io.quarkus.devtools.testing.WrapperRunner.run(WrapperRunner.java:70)
        at io.quarkus.devtools.testing.codestarts.QuarkusCodestartTest.buildProject(QuarkusCodestartTest.java:157)
        at io.quarkus.devtools.testing.codestarts.QuarkusCodestartTest.buildAllProjects(QuarkusCodestartTest.java:145)
        at io.quarkiverse.quinoa.it.QuinoaCodestartTest.testBuild(QuinoaCodestartTest.java:30)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:727)
        at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
        at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:156)
        at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:147)
        at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:86)
        at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(InterceptingExecutableInvoker.java:103)
        at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.lambda$invoke$0(InterceptingExecutableInvoker.java:93)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
        at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:92)
        at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:86)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$7(TestMethodTestDescriptor.java:217)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:213)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:138)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:68)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:151)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
        at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
        at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:147)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:127)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:90)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:55)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:102)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:54)
        at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:114)
        at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:86)
        at org.junit.platform.launcher.core.DefaultLauncherSession$DelegatingLauncher.execute(DefaultLauncherSession.java:86)
        at org.apache.maven.surefire.junitplatform.LazyLauncher.execute(LazyLauncher.java:50)
        at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.execute(JUnitPlatformProvider.java:184)
        at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:148)
        at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:122)
        at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
        at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
        at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
        at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)

[INFO]
[INFO] Results:
[INFO]
[ERROR] Errors:
[ERROR]   QuinoaCodestartTest.testBuild:30 » IllegalThreadState process hasn't exited
[ERROR]   QuinoaRootPathTest.testUIIndex » Runtime java.lang.reflect.InvocationTargetException
[ERROR]   QuinoaUIAngularTest.testUIIndex » Runtime java.lang.reflect.InvocationTargetException
[ERROR]   QuinoaUILitTest.testUIIndex » Runtime java.lang.reflect.InvocationTargetException
[ERROR]   QuinoaUIReactJustBuildTest.testIndexFromResources » Runtime java.lang.reflect.InvocationTargetException
[ERROR]   QuinoaUIReactTest.testUIIndex » Runtime java.lang.reflect.InvocationTargetException
[ERROR]   QuinoaUIVueTest.testUIIndex » Runtime java.lang.reflect.InvocationTargetException
[ERROR]   QuinoaUIYarnTest.testUIIndex » Runtime java.lang.reflect.InvocationTargetException
[INFO]
[ERROR] Tests run: 18, Failures: 0, Errors: 8, Skipped: 9
[INFO]
[WARNING] Corrupted channel by directly writing to native stream in forked JVM 1. See FAQ web page and the dump file /mnt/c/Users/shivam sharma/Desktop/Open-Source/quarkus-quinoa/integration-tests/target/surefire-reports/2023-05-03T06-46-14_768-jvmRun1.dumpstream
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Quarkus - Quinoa - Parent 999-SNAPSHOT:
[INFO]
[INFO] Quarkus - Quinoa - Parent .......................... SUCCESS [ 39.974 s]
[INFO] Quarkus - Quinoa - Deployment - Testing ............ SUCCESS [ 22.408 s]
[INFO] Quarkus - Quinoa - Runtime ......................... SUCCESS [01:07 min]
[INFO] Quarkus - Quinoa - Deployment ...................... SUCCESS [  01:02 h]
[INFO] Quarkus - Quinoa - Testing ......................... SUCCESS [01:12 min]
[INFO] Quarkus - Quinoa - Documentation ................... SUCCESS [01:23 min]
[INFO] Quarkus - Quinoa - Integration Tests ............... FAILURE [49:17 min]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:56 h
[INFO] Finished at: 2023-05-03T08:41:27+05:30
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.0.0:test (default-test) on project quarkus-quinoa-integration-tests:
[ERROR]
[ERROR] Please refer to /mnt/c/Users/shivam sharma/Desktop/Open-Source/quarkus-quinoa/integration-tests/target/surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :quarkus-quinoa-integration-tests

```

**Comment by melloware** [comment]
Maybe @ia3andy knows what that error is?

**Pull request merged**